packages/yarnpkg-core/sources/Configuration.ts

```diff
@@ -929,6 +965,16 @@ export class Configuration {
     return this.values.get(key) as T;
   }
 
+  getSecret<T = any>(key: string) {
```

Comment:
Since it gets everything *but* secrets, I think the name should be more explicit. What about `getSafe` or `getForDisplay`?

Reply:
I'll go with `getForDisplay`. 👍